src/Database/PostgreSQL/PQTypes/SQL/Builder.hs

```diff
@@ -664,16 +692,25 @@ sqlWhereIsNULL col = sqlWhere $ col <+> "IS NULL"
 sqlWhereIsNotNULL :: (MonadState v m, SqlWhere v) => SQL -> m ()
 sqlWhereIsNotNULL col = sqlWhere $ col <+> "IS NOT NULL"
 
+-- | Run monad that joins all conditions with 'AND' operator.
+--
+-- When no conditions are given, the result is 'TRUE'.
+sqlAll :: State SqlWhereAll () -> SQL
```

Comment:
What is it for? Apart from being used in `sqlWhereAny`. Tests?

Because `sqlWhere . sqlAll` is redundant.

Reply:
Well for nesting `AND`s in `OR`'s and parity with `sqlAny`. How else are you going to write `(a OR (b AND c))`?

I mean, you could use `sqlWhereAny`, but if you want to go with `sqlWhere . sqlAny $ ...`, then at one point you have to use `sqlWhere . sqlAll $ ...` inside.

Reply:
Ah, I see. The comment you added to the code makes it clear, thanks.